test/client/fixtures/sandbox/code-instrumentation/getters-test.js

```diff
@@ -134,36 +134,43 @@ test('document.referrer', function () {
 
 if (browserUtils.isWebKit) {
     test('url in stylesheet properties', function () {
-        var el       = document.createElement('div');
-        var url      = 'http://google.com/image.png';
-        var proxyUrl = urlUtils.getProxyUrl(url);
+        var el                 = document.createElement('div');
+        var url                = 'http://google.com/image.png';
+        var proxyUrl           = urlUtils.getProxyUrl(url);
+        var quote              = (function () {
+            var div = document.createElement('div');
+
+            div.style.backgroundImage = 'url(http://example.com/img.jpg)';
+
+            return div.style.backgroundImage.match(/url\((.*)http:\/\/example.com\/img.jpg/)[1];
+        })();
+        var buildExpectedValue = function (value) {
+            return 'url(' + quote + value + quote + ')';
+        };
```

Comment:
Just name it `getExpected` as I suggested previously. There is nothing to `build` here